index.html

```diff
@@ -23,8 +23,54 @@
         <!-- CONTAINER JEST DLA KAŻDEJ SEKCJI OPRÓCZ MODALI -->
         <section class="header">
             <div class="container"></div>
-            <!-- KOMENTARZ DO BETA -->
             <div class="mobile-menu"></div>
         </section>
+        <section class="howitworks">
+            <div class="container">
+                <h2 class="howitworks-title h2-mobile">
+                    To order your fruit basket, simply follow these easy steps
+                </h2>
+                <p class="p-all howitworks-text">
+                    Our baskets are assembled with care and delivered straight
+                    to your doorstep, so you can enjoy the taste of fresh fruit
+                    without ever leaving your home. Whether you're looking for a
+                    healthy snack or a thoughtful gift, our fruit baskets are
+                    the perfect choice.
+                </p>
+                <ul class="howitworks-fieldset">
+                    <li class="howitworks-field">
+                        <button class="howitworks-steps-btn">First Step</button>
```

Comment:
Technicznie rzecz biorąc to napis "First Step" to step powinnień być z małej litery i powinna ona zostać powiększona w css (text-transform).